tests/auto/desktop-qt5/newviewrequest/tst_newviewrequest.qml

```diff
@@ -63,14 +69,14 @@ Item {
             verify(MyScript.waitLoadFinished(mozView))
             compare(mozView.title, "NewWinExample")
             verify(MyScript.wrtWait(function() { return (!mozView.painted); }))
+            mozView.loadFrameScript("chrome://tests/content/testHelper.js")
```

Comment:
This is too late...Could go to ViewComponents.qml:

```
Component.onCompleted: loadFrameScript("chrome://tests/content/testHelper.js")
```

This way all tests that are using dynamically allocated WebView would get it. mozview_p itself takes care of loading it after the view is initialized.